### ISSUE TITLE
DirectXCollision.h about time RH is added?

### ISSUE THREAD

**Issue by cpyburn** [Microsoft/DirectX-Graphics-Samples#199]
This is more of a request than an issue, but maybe it should be an "issue" especially when you are creating the mini-engine in the right-handed coordinate system.  As quoted below

> // Author:  James Stanard 
> // Oh, and we use right-handed coordinate systems because that's what you learned in your Linear Algebra, Calculus, and
> // Physics classes.
> //
> // Peace,
> //   James 

Isn't it about time to implement RH frustum and collision into DirectXCollision.h if you are going to include it with directX?  I have already written my own, because I came from an XNA background which was right-handed, which kind of baffles me why right-handed wasn't already included since it is what XNACollision turned into.

> Note that the XNACollision types are now part of DirectXMath in DirectXCollision.h--no .cpp file. – Chuck Walbourn Jul 28 '15 at 5:59

**Comment by walbourn** [comment]
Mostly it was an oversight, honestly. DirectX samples have historically used left-handed coordinates exclusively. If you have a proposed fixed, then feel free to propose a pull request to the [DirectXMath](https://github.com/Microsoft/DirectXMath) GitHub project. This issue is already tracked [there](https://github.com/Microsoft/DirectXMath/issues/3).

**Comment by cpyburn** [comment]
Unfortunately, I implemented my RH Frustum and Bounding collisions differently than DirectXCollision.h implemented so they are not fully compatible or contain the same methods, for instance I only care about disjoint in my game, so it is all I implemented.  I also made a method CreateFromPoint instead of CreateFromPoints to avoid double iterations.  I may take on the task at a later date since it looks like most the code has already been reviewed in that thread you posted.
